src/functions.php

```diff
@@ -42,14 +42,17 @@ function parallel(callable ...$callbacks): array
 /**
  * @param string|array<string>  $command
  * @param array<string, string> $environment
+ * @param (callable(string, string, Process) :void)|null $callback
  */
 function exec(
     string|array $command,
     ?string $workingDirectory = null,
     array $environment = [],
     bool $tty = false,
+    bool $pty = true,
```

Comment:
At some point, we'll need to document theses variables, because it'n not so obvious